src/components/App/index.tsx

```diff
@@ -12,6 +12,15 @@ import '../custom-els/LoadingSpinner';
 // This is imported for TypeScript only. It isn't used.
 import Compress from '../compress';
 
+const compressPromise = import(
+  /* webpackChunkName: "main-app" */
+  '../compress',
+);
+const offlinerPromise = import(
+  /* webpackChunkName: "offliner" */
+  '../../lib/offliner',
+);
```

Comment:
I moved the service worker stuff into a lazy-loaded module to keep our first interaction size down.